vocabularies/VDM.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.01">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Alias="Core" Namespace="Org.OData.Core.V1" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="com.sap.vocabularies.VDM.v1" Alias="VDM">
+      <Annotation Term="Core.Description">
+        <String>Terms for the Virtual Data Model</String>
+      </Annotation>
+      <Annotation Term="Core.Description" Qualifier="Published">
+        <String>2021-06-18 © Copyright 2021 SAP SE. All rights reserved</String>
+      </Annotation>
+      <Annotation Term="Core.Links">
+        <Collection>
+          <Record>
+            <PropertyValue Property="rel" String="latest-version" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/VDM.xml" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="alternate" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/VDM.json" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="describedby" />
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/master/vocabularies/VDM.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="SAPObjectNodeType" Type="VDM.SAPObjectNodeTypeType" Nullable="false" AppliesTo="EntityType Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The SAP Object Node Type represented by the annotated entity type, or referenced by the annotated property." />
+        <Annotation Term="Core.Description">
+          <String>SAP Object Node Types describe the structure of SAP Object Types.
+            
+SAP Object Types are a generalization of Business Object, Technical Object, Configuration Object, and Analytical Object.</String>
+        </Annotation>
+      </Term>
+
+      <ComplexType Name="SAPObjectNodeTypeType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Information about an SAP Object Node Type" />
+        <Property Name="Name" Type="Edm.String" Nullable="false">
```

Comment:
Is there a namespace concept for SAP object node types? For example, if a partner introduces new types, must their have their own namespace? Perhaps this should be a QualifiedName rather than just an Edm.String.

Reply:
This question goes to the SOT/SONT experts